src/table/table-body.jsx

```diff
@@ -183,10 +183,11 @@ const TableBody = React.createClass({
 
     return React.Children.map(this.props.children, (child) => {
       if (React.isValidElement(child)) {
+        const {selected} = child.props;
         const props = {
           displayRowCheckbox: this.props.displayRowCheckbox,
           hoverable: this.props.showRowHover,
-          selected: this._isRowSelected(rowNumber),
+          selected: selected !== undefined ? selected : this._isRowSelected(rowNumber),
```

Comment:
not sure what you want, do you mean `selected !== undefined && selected || this._isRowSelected(rowNumber),` instead?

I've not put `selected || this._isRowSelected(rowNumber)` because if `selected` is `false`, this is an explicit choice (not undefined) and we should respect it.